tests/appsec/test_traces.py

```diff
@@ -15,9 +15,14 @@
 @released(golang="v1.34.0-rc.4", dotnet="1.29.0", java="?", nodejs="?", php="?", python="?", ruby="?")
 class Test_AppSecEventSpanTags(BaseTestCase):
     def test_appsec_event_span_tags(self):
-        """Spans with AppSec events should have the general AppSec span tags, along with the appsec.event and _sampling_priority_v1 tags"""
+        """
+        Spans with AppSec events should have the general AppSec span tags, along with the appsec.event and
+        _sampling_priority_v1 tags
+        """
 
         @missing_feature(library="ruby", reason="can't report user agent with dd-trace-rb")
+        @missing_feature(library="dotnet",
```

Comment:
This decorator should be on the test function (`test_appsec_event_span_tags`), not here

Reply:
Oops, my bad, now fixed.